iothub/device/src/HsmAuthentication/HttpHsmSignatureProvider.cs

```diff
@@ -2,19 +2,27 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Linq;
+using System.Net.Http;
 using System.Text;
 using System.Threading.Tasks;
+#if NETSTANDARD2_0
+using Microsoft.Azure.Devices.Client.HsmAuthentication.Transport;
+#endif
 using Microsoft.Azure.Devices.Client.TransientFaultHandling;
 
 namespace Microsoft.Azure.Devices.Client.HsmAuthentication
 {
     class HttpHsmSignatureProvider : ISignatureProvider
     {
-        const string DefaultApiVersion = "2018-06-28";
-        const SignRequestAlgo DefaultSignRequestAlgo = SignRequestAlgo.HMACSHA256;
-        const string DefaultKeyId = "primary";
-        readonly string apiVersion;
-        readonly HsmHttpClient httpClient;
+        private const string DefaultApiVersion = "2018-06-28";
```

Comment:
>2018-06-28 [](start = 50, length = 10)

Is this API version is not affected by the pre GA post GA version changes?

Reply:
This is the GA version, pre GA we will use the same version and post GA if there are changes will have to publish a new package that uses the new version. I think this is similar to sdk version for IotHub.